statchatui/src/Compare/Compare.jsx

```diff
@@ -21,10 +21,53 @@ const Compare = ({ signedInUserData }) => {
   const [isLoading, setIsLoading] = useState(false);
   const [otherUserStats, setOtherUserStats] = useState(null);
   const [signedInUserStats, setSignedInUserStats] = useState(null);
+  const [userQueue, setUserQueue] = useState([]);
+  const [showStats, setShowStats] = useState(null);
+  const storedUserQueue = sessionStorage.getItem("userQueue");
```

Comment:
I suggest moving `userQueue` into a constant at the top of the module. This will help you avoid making typos and make it clear that it's a constant string key.
```suggestion
  const storedUserQueue = sessionStorage.getItem(USER_QUEUE);
```